lib/urlwatch/command.py

```diff
@@ -136,6 +136,28 @@ def test_filter(self):
         # (ignore_cached) and we do not want to store the newly-retrieved data yet (filter testing)
         return 0
 
+    def run_job(self):
+        job = self._find_job(self.urlwatch_config.run_job)
+        if job is None:
+            # Split the Input and Search for Jobs.
+            job_items = [self._find_job(item) for item in self.urlwatch_config.run_job.split(',')]
```

Comment:
URLs may contain commas. It's probably not a good idea to use commas to separate items.

Reply:
I can add a util Funktion to Look for escaped commas and/or ignore commas that are in a Quote (or escaped them before the split). That would bei usefuel for many Others parts too.

Reply:
I suggest supporting specifying the option multiple times instead. It'd be much nicer than dealing with messy escapes.